nodejs/lib/util/http.ts

```diff
@@ -0,0 +1,236 @@
+import Platform from 'platform';
+import * as Utils from '../../../common/lib/util/utils';
+import Defaults from '../../../common/lib/util/defaults';
+import ErrorInfo from '../../../common/lib/types/errorinfo';
+import request, { Options as RequestOptions, RequestResponse } from 'request';
+import { ErrnoException, IHttp, PathParameter, RequestCallback, RequestParams } from '../../../common/types/http';
+import HttpMethods from '../../../common/constants/HttpMethods';
+
+type Rest = any;
+type Realtime = any;
+
+const msgpack = Platform.msgpack;
+const noop = function() {};
```

Comment:
While I recognise most programmers are familiar with the concept of a 'no-op', it's also earnt itself a [disambiguation page on Wikipedia](https://en.wikipedia.org/wiki/Noop_(disambiguation)). Also, for what this variable is actually doing, I think a name like `doNothing` would be more explicit and perhaps more self-explanatory at the sites where it is used.